Config.cmake

```
@@ -68,11 +68,21 @@ set(ODBFLAGS
   --include-with-brackets
   --default-pointer "std::shared_ptr")
 
+# Set CXX flags
 set(CMAKE_CXX_FLAGS "-W -Wall -Wextra -pedantic\
   -std=c++14 \
   -DDATABASE_${DATABASE_U} \
   -DBOOST_LOG_DYN_LINK")
 
+# Gold is the primary linker 
+if(NOT DEFINED CODECOMPASS_LINKER)
+    set(CODECOMPASS_LINKER "/usr/bin/gold")
```

```suggestion
    set(CODECOMPASS_LINKER "gold")
```

So this would be the "magic", depending on the OS to locate the binary?

---

Yes. Yes it was :(